app/assets/javascripts/dialog_field_refresh.js

```diff
@@ -1,16 +1,20 @@
 /* global miqInitSelectPicker miqSelectPickerEvent miqSparkle miqSparkleOn */
 
 var dialogFieldRefresh = {
+  unbindAllPreviousListeners: function() {
+    $(document).off('autoRefresh');
```

Comment:
@eclarizio is 'autoRefresh' our trigger event name - like the old 'message' that was pulled out? Or is it specific to JS?